src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.interfaces.FigureInterface;
+import java.util.Random;
+
+public abstract class Figure implements FigureInterface {

```

Comment:
we also want to draw our figure:
> behaviour - we can obtain the area of any figure and are able to draw it. To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override toString() method for this).

we can create and implement one more interface for this purpose